input/chains/penumbra-1.json

```diff
@@ -314,7 +348,7 @@
     "transfer/channel-4/factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos": 800000000096,
     "transfer/channel-2/ausdy": 800000000095,
     "transfer/channel-4/factory/osmo1em6xs47hd82806f5cxgyufguxrrc7l0aqx7nzzptjuqgswczk8csavdxek/alloyed/allUSDT": 700000000000,
-    "transfer/channel-1/adydx": 600000000000,
+    "transfer/channel-9/adydx": 600000000000,
```

Comment:
Not sure why this value changed... shouldn't that still be 1, or else 16, the new channel?

Reply:
The value changed because the channel-1 dydx is bricked, but the new value is wrong, it should be 16